### PR TITLE
Resolving findings

### PR DESCRIPTION
**Pull request by mrdpopov**
## Purpose

* add missing dependency
* add docs for the expected system configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Microservice to allow 3rd party discovery services to determine the availability
 
 ## Additional information
 
+The expectation is that the **mode of issuance "serial"** and **nature of content "journal" and "newspaper"** are present in the target system. The module uses these values to qualify an instance as a periodical.
```
Comment (mreno-EBSCO) [inline README.md:14]: I guess this is fine for now, but I am concerned that these values are taken from controlled vocabulary and that an institution may want to translate these values to another language. These mappings should probably be externalized, settable perhaps via an RTAC settings app in FOLIO.

File: descriptors/ModuleDescriptor-template.json [hunk 2]
```diff
@@ -54,7 +54,8 @@
       "displayName": "RTAC - all permissions",
       "description": "Entire set of permissions needed to use RTAC",
       "subPermissions": [
-        "rtac.batch.post"
+        "rtac.batch.post",
+        "rtac.holdings.item.get"
```
Comment (mreno-EBSCO) [inline descriptors/ModuleDescriptor-template.json:58]: We also need the actual `rtac.holdings.item.get` permission to be created as a permission set, similar to `rtac.batch.post` on line 48.

File: README.md (hunk 1 above)
Comment (mrdpopov) [reply README.md:14]: This documentation was added as it was discussed with @craigmcnally

File: descriptors/ModuleDescriptor-template.json (hunk 2 above)
Comment (mrdpopov) [reply descriptors/ModuleDescriptor-template.json:63]: done.

**Review by mreno-EBSCO — APPROVED** [review]

**Pull request merged**